dap_prinz_green_jobs/notebooks/between_measures_analysis-Aug23.py

```diff
@@ -1,25 +1,13 @@
-# ---
-# jupyter:
```

Comment:
@india-kerle I realised I pushed the `.py` file not the `.ipynb` file! I'll replace them